.env.example

```
@@ -1,5 +1,6 @@
-# The bundle identifier of your app
-FASTLANE_APP_IDENTIFIER=club.ntut.tattoo
+# Store identities for the existing classic TAT listings
+IOS_BUNDLE_IDENTIFIER=com.npc.tatFlutter
+ANDROID_PACKAGE_NAME=club.ntut.npc.tat
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANDROID_PACKAGE_NAME key should go before the IOS_BUNDLE_IDENTIFIER key

(UnorderedKey)